topology/National Center for High-performance Computing/NCHC/NCHC-CMS-ITP.yaml

```diff
@@ -0,0 +1,44 @@
+Production: false
+SupportCenter: GGUS
+
+GroupDescription: Opportunistic resource from CMS project.
+GroupID: 
+
+Resources:
+  NCHC-CMS-CE:
+    Active: false
+    Description: Opportunistic resource from CMS project.
+    ID: 
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Chun-Yu Lin
+          ID: 383ae4982bef24cc546b47c4e4f2e52b57e32da6
+
+      Security Contact:
+        Primary:
+          Name: Chun-Yu Lin
+          ID: 383ae4982bef24cc546b47c4e4f2e52b57e32da6
+
+    FQDN: ui.grid.nchc.org.tw
+
+    Services:
+      CE:
+        Description: HTCondor-CE
+
+    VOOwnership:
+      CMS: 100
+      IGWN: 200
+     
+    # WLCGInformation:
+    #   APELNormalFactor: 0.0
+    #   AccountingName: <name>
+    #   HEPSPEC: 0
+    #   InteropAccounting: true
+    #   InteropBDII: true
+    #   InteropMonitoring: true
+    #   KSI2KMax: 0
+    #   KSI2KMin: 0
+    #   StorageCapacityMax: 0
+    #   StorageCapacityMin: 0
+    #   TapeCapacity: 0
```

Comment:
@chunyulin if you're contributing to CMS, I suspect that they'll need these numbers unless you are uploading APEL accounting data directly to WLCG. Normally, OSG sites will publish information to the GRACC (your CE should do this automatically) and then the GRACC sends monthly APEL numbers to the WLCG.

Reply:
I will start with IGWN first and should be OK w/o this.